### PR TITLE
Version bump to 2.0.2

### PR DESCRIPTION
**Pull request by albingeorge**
### Added
- Support for creating and fetching Transfers
- Support for creating Reversals on transfers

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -75,7 +74,7 @@ Changelog for Razorpay-PHP SDK. Follows [keepachangelog.com](http://keepachangel
 - Initial Release
 
 
-[Unreleased]: https://github.com/razorpay/razorpay-php/compare/2.0.1...HEAD
```
Comment (amitmahbubani) [inline CHANGELOG.md]: Add link for `2.0.2` changes below

**Comment by amitmahbubani** [comment]
Update the composer require version to 2.x in the README:

```
{
    "require": {
        "razorpay/razorpay": "1.*"
    }
}
```

File: CHANGELOG.md (hunk 1 above)
Comment (albingeorge) [inline CHANGELOG.md]: Added

**Comment by albingeorge** [comment]
@amitmahbubani Made the above changes.

**Pull request merged**